string type
geometry_msgs/Point[] points
float64[] normVals
bool hasNext
bool swapped
geometry_msgs/Point[] newTrajPoints
